uimodule/webapp/Firebase.js

```diff
@@ -26,13 +33,18 @@ sap.ui.define([
             usersRef.on("value", snapshot => {
```

Comment:
hung notation

Reply:
It should not be in Firebase.js file I guess